libnd4j/include/helpers/EigenValsAndVecs.h

```diff
@@ -79,6 +79,14 @@ class EigenValsAndVecs {
 
 };
 
+/**
+ * @brief Compute the eigenvalues and eigenvectors of a square matrix
+ * 
+ * @param input square matrix {n,n}
+ * @param vals eigenvalues {n,2}
+ * @param vecs eigenvectors {n,n,2}
+ */
+void eig(const NDArray& input, NDArray& vals, NDArray& vecs);
 
```

Comment:
Is this empty on purpose?

Reply:
its header declaration. I did not understand what is empty there